styles.css

```diff
@@ -93,7 +101,338 @@ ul.navbar-items {
   line-height: 1.5rem;
 }
 
-@media only screen and (max-width: 375px) {
+/* Works section styling */
+
+#works-section {
+  margin: 10px 0;
+  overflow-y: hidden;
+}
+
+.box-container {
+  display: flex;
+  margin: 50px 200px 50px 200px;
+}
+
+.left-section {
+  flex: 1;
+}
+
+.left-section img {
+  float: left;
+  margin-right: 30px;
+  max-width: 100%;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 10px;
+  margin: 0 200px;
+}
+
+.grid-item {
+  position: relative;
+  background-color: #eaeaea;
+  height: 400px;
+}
+
+.first-desc h3 {
+  color: #091e42 !important;
+  font-size: 32px !important;
+  line-height: 44px !important;
+}
+
+.first-desc p {
+  font-size: 16px !important;
+  line-height: 25px;
+  text-align: left;
+}
+
+.first-desc ul li {
+  display: inline-block;
+  list-style: none;
+  border: 1px solid grey;
+  padding: 10px;
+  margin-right: 10px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.first-desc button {
+  background-color: #f55800;
+  border: none;
+  padding: 10px 10px;
+  margin-top: -40px !important;
+  font-family: 'Crete Round', sans-serif !important;
+  font-style: normal !important;
+  font-weight: 700px !important;
+  font-size: 16px !important;
+  line-height: 30px !important;
+  letter-spacing: 0.03em;
+  color: #eaeaea;
+}
+
+.grid-item:nth-child(1) {
+  background-image: url("./img/grid-1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(2) {
+  background-image: url("./img/grid-2.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(3) {
+  background-image: url("./img/grid-3.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(4) {
+  background-image: url("./img/grid-4.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(5) {
+  background-image: url("./img/grid-5.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.grid-item:nth-child(6) {
+  background-image: url("./img/grid-6.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.item-child {
+  position: absolute;
+  bottom: 20px;
+  width: 100%;
+  text-align: left;
+}
+
+.items-details {
+  padding: 50px 50px 10px 10px;
+  text-align: left;
+}
+
+.item-child h3 {
+  color: #ffff;
+  text-align: left;
+  margin-left: 10px;
+  margin-bottom: 0;
+}
```

Comment:
- **[OPTIONAL]** Please 🙏🏽 use the correct font family and styles to be similar to the Figma template 👇🏽

![image](https://github.com/Xghayor/Portfolio/assets/58642949/1d6f9483-eced-4c48-bfe0-cbc391ee2de9)